### DISCUSSION TITLE
Unable to Log In

### DISCUSSION THREAD

**Discussion by mjrascoe** [nfrastack/container-freescout#118] [Q&A]
Hello,

I used the docker-compose.yml for no proxy environment.  I can reach the login page, but am unable to log in with the credentials used in the yml file

Any help would be appreciated, please see my yml file below.  I only modified the port, site URL and timezone

`version: '2'

services:
  freescout-app:
    image: tiredofit/freescout
    container_name: freescout-app
    ports:
    - 80:80
    links:
    - freescout-db
    volumes:
    ### If you want to perform customizations to the source and have access to it, then uncomment this line - This includes modules
    #- ./data:/www/html
    ### Or, if you just want to use Stock Freescout and hold onto persistent files like cache and session use this, one or the other.
    - ./data:/data
    ### If you want to just keep the original source and add additional modules uncomment this line
    - ./modules:/www/html/Modules
    - ./logs/:/www/logs
    environment:
    - CONTAINER_NAME=freescout-app
     - DB_HOST=freescout-db
    - DB_NAME=freescout
    - DB_USER=freescout
    - DB_PASS=freescout
    - SITE_URL=http://192.168.50.144:8582
    - ADMIN_EMAIL=admin@admin.com
    - ADMIN_PASS=freescout
    - ENABLE_SSL_PROXY=FALSE
    - DISPLAY_ERRORS=FALSE
    - TIMEZONE=Asia/Hong_Kong
    restart: always

  freescout-db:
    image: tiredofit/mariadb
    container_name: freescout-db
    volumes:
      - ./db:/var/lib/mysql
    environment:
      - ROOT_PASS=password
      - DB_NAME=freescout
      - DB_USER=freescout
      - DB_PASS=freescout
      - CONTAINER_NAME=freescout-db
    restart: always

  freescout-db-backup:
    container_name: freescout-db-backup
    image: tiredofit/db-backup
    links:
     - freescout-db
    volumes:
      - ./dbbackup:/backup
    environment:
      - CONTAINER_NAME=freescout-db-backup
      - DB_HOST=freescout-db
      - DB_TYPE=mariadb
      - DB_NAME=freescout
      - DB_USER=freescout
      - DB_PASS=freescout
      - DB_DUMP_FREQ=1440
      - DB_DUMP_BEGIN=0000
      - DB_CLEANUP_TIME=8640
      - COMPRESSION=BZ
      - MD5=TRUE
    restart: always`

![fsloginerror](https://user-images.githubusercontent.com/28326324/220263229-9d250349-1521-44ac-a985-3e5c33db0da0.png)

**Comment by Chadnic** [comment]
Firstly your port is wrong as it doesn't match the port in your URL.

You have:

`ports:`
`- 80:80`

and it should be

`ports:`
`- 8252:80`

for what you show in the site url:

`SITE_URL=http://192.168.50.144:8582/`


You probably have NGINX proxy running that is doing an automatic redirect, but I've seen a ton of problems when the ports are not matched up correctly.  Also if you are using SSL or other things it also can cause no login conditions if the ports are not matched correctly.

One more thing, why not just change the user name and password to something and try it rather than using the default?

It's a place to start!
